Formula/parquet-tools.rb

```diff
@@ -0,0 +1,22 @@
+class ParquetTools < Formula
+  desc "ApacheParquet command-line tools and utilities"
+  homepage "https://parquet.apache.org/"
+  url "https://github.com/apache/parquet-mr.git", :tag => "apache-parquet-1.8.1", :revision => "4aba4dae7bb0d4edbcf7923ae1339f28fd3f7fcf"
+
+  head "https://github.com/apache/parquet-mr.git"
+
+  depends_on "maven" => :build
+
+  def install
+    ENV.java_cache
+    cd "parquet-tools" do
+      system "mvn", "clean", "package", "-Plocal"
+      libexec.install "target/parquet-tools-#{version}.jar"
+      bin.write_jar_script libexec/"parquet-tools-#{version}.jar", "parquet-tools"
+    end
+  end
+
+  test do
+    system "parquet-tools", "cat", "-h"
```

Comment:
Please also call the test executable with a full path, i.e.:

```ruby
system bin/"parquet-tools", "cat", "-h"
```
Or:
```ruby
system "#{bin}/parquet-tools", "cat", "-h"
```
Using the current test as an example.